docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds when container images or charts are
```

Comment:
The precision of this statement could be better. It responds to _notifications_ of images or charts having been pushed. The role of the webhook request is an important detail. Without that, this seems like magic.